src/main/java/io/anserini/qa/passage/PassageScorer.java

```diff
@@ -0,0 +1,7 @@
+package io.anserini.qa.passage;
+
+import java.util.List;
+
+public interface PassageScorer {
+  void score(List<String> sentences, String index, String output, Context context) throws Exception;
+}
```

Comment:
Add method:

`List<ScoredPassage> extractTopPassages`

The internal implementation of `PassageScorer` keeps track of the top k in a heap.

Create new object: a `ScoredPassage` is just Passage (sentence String) with a score.